DEVELOP.md

```diff
@@ -3,11 +3,14 @@
 If you've made it this far, **thank you**! We appreciate your contribution, and hope that this document helps you along the way. If you have any questions or problems, don't hesitate to [file an issue](https://github.com/primer/css/issues/new).
 
 ## Structure
-Primer CSS is published to [npm] as [@primer/css]. Each of Primer CSS's "modules" lives in a subfolder under `src/` with an `index.scss` in it. Generally speaking, the styles are divided into three primary themes:
+Primer CSS is published to [npm] as [@primer/css]. Each of Primer CSS's components lives in a subfolder under `src/` with an `index.scss` in it. Generally speaking, the styles are divided into three primary themes:
```

Comment:
The `three primary themes` isn't really accurate anymore. Not sure how to call these "non component" folders.. 😅  maybe:

```suggestion
Primer CSS is published to [npm] as [@primer/css]. Each of Primer CSS's components lives in a subfolder under `src/` with an `index.scss` in it. You will also find a few folders that contain additional styles:
```